src/Symfony/Component/Messenger/Transport/Doctrine/DoctrineReceiver.php

```diff
@@ -46,6 +47,10 @@ public function get(): iterable
     {
         try {
             $doctrineEnvelope = $this->connection->get();
+        } catch (RetryableException $exception) {
+            // Do nothing when RetryableException occurs.
+            // Problem with concurent consumers and Database Deadlocks
```

Comment:
typo: concurrent and lower case database deadlocks

Reply:
updated